features/support/paths.rb

```diff
@@ -286,10 +286,6 @@ def path_to(page_name, *args)
       plan = Plan.find_by_name!($1)
       edit_polymorphic_path([:admin, plan])
 
-    when 'the latest transactions page'
```

Comment:
I removed it because it already exists in this same file in:
https://github.com/3scale/porta/blob/297411f4c5b12086b063828dd8549cbc6ea0a472/features/support/paths.rb#L610-L611